Service Connector  Hub/requirements.txt

```diff
@@ -1,4 +1,5 @@
 fdk
 datetime
 requests
-oci
\ No newline at end of file
+oci
+yaml
```

Comment:
line break at the end of file

Reply:
Good catch. It didn't exist before, and I missed adding it. Thanks.